beacon-chain/p2p/gossip_scoring_params.go

```diff
@@ -273,6 +314,21 @@ func scoreDecay(totalDurationDecay time.Duration) float64 {
 	return math.Pow(decayToZero, 1/float64(numOfTimes))
 }
 
+func decayThreshold(decayRate, rate float64) (float64, error) {
```

Comment:
Despite being non-public, can we still have a comment describing what function does? Otherwise, one need to review and understand the code (which is not hard, but definitely not as easy at it can be explained in a function comment).